nilearn/glm/contrasts.py

```diff
@@ -411,19 +419,32 @@ def compute_fixed_effects(contrast_imgs, variance_imgs, mask=None,
         masker = NiftiMasker(mask_img=mask).fit()
 
     variances = masker.transform(variance_imgs)
-    contrasts = masker.transform(contrast_imgs)
+    contrasts = np.array([masker.transform(contrast_img)
+                          for contrast_img in contrast_imgs])
+
+    if dofs is not None:
```

Comment:
Would it work to set dofs to False by default and then put all the new code including the new return statement under `if dofs:`
and then put the old code and return outside this if statement and add a deprecation warning saying to set dofs to a value or None?